babel/messages/catalog.py

```diff
@@ -840,3 +855,16 @@ def _key_for(self, id, context=None):
         if context is not None:
             key = (key, context)
         return key
+
+    def is_eq(self, other):
```

Comment:
Not really happy with having these `is_eq`, but haven't come up with anything better so far.

Reply:
Maybe a better name (as mused upon around the other `is_eq`) (and a docstring would be nice)...